.prettierignore

```diff
@@ -70,8 +70,6 @@ app/javascript/styles/mastodon/reset.scss
 # Ignore Javascript pending https://github.com/mastodon/mastodon/pull/23631
 *.js
 *.jsx
```

Comment:
Should we un-ignore the .js. & .jsx files as well now as well?

Or in another huge commit reformatting all of those?

Reply:
Yeah, there were included before, but the diff was too large, so they were removed to have just the TS files as a first step